docs/server.md

```diff
@@ -16,27 +16,65 @@ mxnet-model-server --models squeezenet=https://s3.amazonaws.com/model-server/mod
 
 With the command above executed, you have MMS running on your host, listening for inference requests.
 
-To test it out, download a [cute picture of a kitten](https://www.google.com/search?q=cute+kitten&tbm=isch&hl=en&cr=&safe=images) and name it `kitten.jpg`. Then run the following `curl` command to post an inference request with the image. In the example below both of these steps are provided.
+To test it out, you will need to open a new terminal window next to the one running MMS. Then we will use `curl` to download one of these [cute pictures of a kitten](https://www.google.com/search?q=cute+kitten&tbm=isch&hl=en&cr=&safe=images) and curl's `-o` flag will name it `kitten.jpg` for us. Then we will `curl` a `POST` to the MMS predict endpoint with the kitten's image. In the example below, both of these steps are provided.
 
 ```bash
-wget -O kitten.jpg \
+curl -o kitten.jpg \
   https://upload.wikimedia.org/wikipedia/commons/8/8f/Cute-kittens-12929201-1600-1200.jpg
 curl -X POST http://127.0.0.1:8080/squeezenet/predict -F "data=@kitten.jpg"
 ```
 
-Then check the [API description](http://127.0.0.1:8080/api-description). For info on other endpoints check out the [REST API documentation](rest_api.md).
+![kitten](https://upload.wikimedia.org/wikipedia/commons/8/8f/Cute-kittens-12929201-1600-1200.jpg)
 
-For more models, check out the [model zoo](model_zoo.md).
+The predict endpoint will return a prediction response in JSON. Each of the probabilities are percentages, and the classes are coming from a `synset` file included inside the model archive which holds the thousand ImageNet classes this model is matching against. It will look something like the following result, where the 0.94 result is a 94% probable match with an Egyptian cat:
+
+```
+{
+  "prediction": [
+    [
+      {
+        "class": "n02124075 Egyptian cat",
+        "probability": 0.9408261179924011
+      },
+      {
+        "class": "n02127052 lynx, catamount",
+        "probability": 0.055966004729270935
+      },
+      {
+        "class": "n02123045 tabby, tabby cat",
+        "probability": 0.0025502564385533333
+      },
+      {
+        "class": "n02123159 tiger cat",
+        "probability": 0.00034320182749070227
+      },
+      {
+        "class": "n02123394 Persian cat",
+        "probability": 0.00026897044153884053
+      }
+    ]
+  ]
+}
+```
+You will see this result in the response to your `curl` call to the predict endpoint, and in the server logs in the terminal window running MMS.
```

Comment:
let's omit the bit about server logs. it's true right now, but we should change the logging so that it isn't (at least, not at default log level)